pkg/runner/flags.go

```diff
@@ -62,6 +66,12 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	fs.StringVar(&cfg.TLSCertFile, "tls-cert-file", "", "TLS Certificate file for web server, leave blank to disable TLS.")
 	fs.StringVar(&cfg.TLSKeyFile, "tls-key-file", "", "TLS Key file for web server, leave blank to disable TLS.")
 
+	// Multi-tenancy flags.
+	// TODO(Harkishen): update the docs in docs/cli.md
+	fs.BoolVar(&cfg.EnableMultiTenancy, "multi-tenancy", false, "Use multi-tenancy mode in Promscale.")
```

Comment:
I would have a separate bool for 
fs.BoolVar(&cfg.EnableDataWithoutTenant, "multi-tenancy-no-tenant-name", false, "If multi-tenancy is on, allow data with no tenant")

Reply:
that will be confusing for user. They will be confused whether to apply true to both `multi-tenancy` and `multi-tenancy-no-tenant-name` or just the later. So I think we should have a bool saying `skip-tenant-validaton` whose purpose is self-explanatory.

Reply:
(Note to myself) TODO

Reply:
I think it's better to follow https://github.com/timescale/promscale/pull/550#discussion_r612686267 for this case.